test/ziggurat/messaging/producer_test.clj

```diff
@@ -227,39 +212,31 @@
               (is (= (get message-from-mq :retry-count 0) @retry-count))
               (producer/retry message-from-mq)))
           (let [message-from-mq (rmq/get-msg-from-dead-queue "default")]
-            (is (= expected-message-payload message-from-mq))))))))
+            (is (= (bytes-to-str expected-message-payload) (bytes-to-str message-from-mq)))))))))
 
 (deftest retry-with-exponential-backoff-test
   (testing "message will publish to delay with retry count queue when exponential backoff enabled"
     (with-redefs [ziggurat-config (constantly (assoc (ziggurat-config)
                                                      :retry {:count   5
                                                              :enabled true
                                                              :type    :exponential}))]
-      (testing "message with no retry count will publish to delay queue with suffix 1"
```

Comment:
instead of deleting, we can probably change this to mean: "message with the value of retry-count equal to the configured retry-count value, will publish to delay queue with suffix 1"